src/main/java/uk/gov/cshr/civilservant/domain/CivilServant.java

```diff
@@ -41,6 +42,10 @@
     @ManyToMany
     private Set<Profession> otherAreasOfWork = new HashSet<>();
 
+    private String lineManagerUid;
```

Comment:
DOD - According to our plan lineManager should be a CivilServant.

https://cshrdigitalandanalysis.atlassian.net/secure/attachment/10227/10227_linemanager.png

We have email crossing boundaries into this service now. I will pick this up with Richard. Please leave it with me.

DOD - Regardless of the above outcome, missing extra fields defiition in the `schema-mysql.sql`

Reply:
Will update. Will also query for line manager email address every time to avoid stale data - a little concerned about performance but only really because we seem to be having some issues. Will update this today or tomorrow pending a little testing.